lib/src/components/NxThreatIndicator/__tests__/NxThreatIndicator.test.tsx

```diff
@@ -4,116 +4,104 @@
  * the terms of the Eclipse Public License 2.0 which accompanies this
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
-import { faCircle } from '@fortawesome/free-solid-svg-icons';
-
-import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
-import 'jest-enzyme';
-
-import NxThreatIndicator from '../NxThreatIndicator';
-import NxFontAwesomeIcon from '../../NxFontAwesomeIcon/NxFontAwesomeIcon';
-import NxToolip from '../../NxTooltip/NxTooltip';
+import NxThreatIndicator, { Props } from '../NxThreatIndicator';
+import { rtlRender, userEvent } from '../../../__testutils__/rtlUtils';
+import { screen } from '@testing-library/react';
+import { allThreatLevelCategories } from '../../../util/threatLevels';
 
 describe('NxThreatIndicator', function() {
-  const getShallowComponent = enzymeUtils.getShallowComponent(NxThreatIndicator, {});
+  const quickRender = rtlRender<Props>(NxThreatIndicator, {});
 
-  it('renders an faCircle NxFontAwesomeIcon with the `nx-threat-indicator` class', function() {
-    const icon = getShallowComponent().find(NxFontAwesomeIcon);
-
-    expect(icon).toHaveClassName('nx-threat-indicator');
-    expect(icon).toExist();
-    expect(icon).toHaveProp('icon', faCircle);
+  it('renders a threat indicator svg', function() {
```

Comment:
The fact that it's an svg is also an implementation detail IMO.  The relevant accessibility check would be that it has `role="img"` (when `presentational` is not set)

Reply:
> The fact that it's an svg is also an implementation detail IMO. The relevant accessibility check would be that it has `role="img"` (when `presentational` is not set)

Changed here: 4c116c4eab2f01f2c1f3089e42a7d0c415cc10c2